### PR TITLE
fix: removed page refresh when nft bid is made

### PR DESCRIPTION
**Pull request by superzordon**
Removed page refreshes when bid placed, NFT sold, NFT sold with unlockable content, and auction closed

### CODE REVIEW

**Comment by maebeam** [comment]
We need to pull fresh data so you can see your bid you've placed

**Comment by lazynina** [comment]
@superzordon we need to handle the case where a user submits a bid from the nft-post-page. It looks like we only update the feed-post's NFT entries when that happens. I'd suggest that we emit an event from feed-post in the subscription to the onHide event for the `PlaceBidModalComponent`. The NFT Post Page component should bind to that event and then perform the following code:
```
this.loading = true;
this.refreshPosts();
```

File: src/app/nft-post-page/nft-post/nft-post.component.ts [hunk 1]
```diff
@@ -218,6 +218,11 @@ export class NftPostComponent {
     this.globalVars.loggedInUser.BlockedPubKeys[blockedPubKey] = {};
   }
 
+  afterNftBidPlaced() {
+    this.loading = true;
+    this.refreshPosts();
```
Comment (lazynina) [inline src/app/nft-post-page/nft-post/nft-post.component.ts:223]: we only need to call `this.refreshBidData()` actually - sorry!

**Review by lazynina — CHANGES_REQUESTED** [review]
after line 585, add `this.nftBidPlaced.emit()` to trigger the nft post page to refresh its bids.

File: src/app/nft-post-page/nft-post/nft-post.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { ChangeDetectorRef, Component } from "@angular/core";
+import {ChangeDetectorRef, Component, ViewChild} from "@angular/core";
```
Comment (lazynina) [inline src/app/nft-post-page/nft-post/nft-post.component.ts:1]:
```suggestion
import { ChangeDetectorRef, Component, ViewChild } from "@angular/core";
```

**Review by lazynina — APPROVED** [review]
nice work!

**Pull request merged**